src/quo2/components/tags/status_tags.cljs

```diff
@@ -45,49 +45,74 @@
+                        (= theme :dark)             colors/success-60)}])
 
 (defn- negative
-  [size theme label]
+  [size theme label blur?]
   [base-tag
    {:size             size
-    :icon             :untrustworthy
+    :icon             :i/negative-state
     :background-color colors/danger-50-opa-10
     :border-color     colors/danger-50-opa-20
     :label            label
-    :text-color       (if (= theme :light)
-                        colors/danger-50
-                        colors/danger-60)}])
+    :text-color       (cond
+                        (= theme :light)           colors/danger-50
+                        ;; The negative tag uses the same color for dark themed and dark blur variant
+                        (or (= theme :dark) blur?) colors/danger-60)}])
+
+(defn- get-color-or-icon-name
+  [{:keys [blur? theme dark-blur-variant light-variant dark-variant]}]
+  (if blur?
+    ;; status tag support only dark blur variant
+    dark-blur-variant
+    (if (= theme :light) light-variant dark-variant)))
 
 (defn- pending
-  [size _ label]
+  [size theme label blur?]
   [base-tag
    {:size             size
-    :icon             :pending
     :label            label
-    :background-color colors/white-opa-5
-    :border-color     colors/white-opa-5
-    :text-color       colors/white-opa-70}])
+    :icon             (get-color-or-icon-name {:blur?             blur?
+                                               :theme             theme
+                                               :dark-blur-variant :i/pending-dark-blur
+                                               :light-variant     :i/pending-light
+                                               :dark-variant      :i/pending-dark})
+    :background-color (get-color-or-icon-name {:blur?             blur?
+                                               :theme             theme
+                                               :dark-blur-variant colors/white-opa-5
+                                               :light-variant     colors/neutral-10
+                                               :dark-variant      colors/neutral-80-opa-40})
+    :border-color     (get-color-or-icon-name {:blur?             blur?
+                                               :theme             theme
+                                               :dark-blur-variant colors/white-opa-5
+                                               :light-variant     colors/neutral-20
+                                               :dark-variant      colors/neutral-80})
+    :text-color       (get-color-or-icon-name {:blur?             blur?
+                                               :theme             theme
+                                               :dark-blur-variant colors/white-opa-70
+                                               :light-variant     colors/neutral-50
+                                               :dark-variant      colors/neutral-40})}])
 
 (defn status-tag
```

Comment:
how about adding some component tests for these? :)

Reply:
Sure, I will add them 👍